modulefiles/OznMonBuild.jet

```diff
@@ -0,0 +1,17 @@
+#%Module################################################
+# Modfule file for oznmon
+########################################################
+export ver=2.0.2
+export FCOMP=mpiifort
+
+export CF=$FCOMP
+export FC=$FCOMP
```

Comment:
how can you set these before loading the compiler?

Reply:
The JET build with this version of OznMonBuild.jet was working.

Reply:
It is working, but not because of this line.

Setting these variables here is doing nothing since the compiler module is overwriting it.

Reply:
We don't need the modulefiles/OznMonBuild.* and modulefiles/RadMonBuild.* modulefiles anymore, please delete all of them. Thanks!

Reply:
Removed as requested.